src/lib/tools.ts

```diff
@@ -0,0 +1,92 @@
+import path from 'path'
+import {promises as fs } from 'fs'
+import _ from 'lodash'
+import copy from 'recursive-copy'
+
+async function copyStripped(srcDir: string, bookStripped: Object, metadataStriped: Object, dstDir: string, paths: string[]): Promise<void> {
+  paths.unshift('.guides/**')
+  paths.unshift('.github/**')
+  await copy(srcDir, dstDir, {
+    filter: paths,
+    overwrite: true,
+    dot: true
+  })
+  const bookJsonPath = path.join(dstDir, '.guides', 'book.json')
+  const metadataPath = path.join(dstDir, '.guides', 'metadata.json')
+  await fs.mkdir(path.join(srcDir, '.guides'), {recursive: true})
+  await fs.writeFile(bookJsonPath, JSON.stringify(bookStripped, undefined, ' '))
+  await fs.writeFile(metadataPath, JSON.stringify(metadataStriped, undefined, ' '))
+}
+
+function traverseBook(book: any, sections: string[]): any {
+  const sectionName = sections.shift()
+  const section = _.find(book.children, {title: sectionName})
```

Comment:
what do you think about strip names here and make search case insensitive?

Reply:
do not think that is required for now